interview-questions.md

```diff
@@ -15,54 +15,62 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
-
-  Researched answer:
+  <!-- Your answer: 
+    Git is software and Github is hosting site for code. -->
+  <!-- Researched answer:  
+    Git is a System that lets you manage and organize code. Github is a the cloud based host site that lets you manage GIT repositories. --> 
 
 
 
-2. Which JavaScript operators will return a Boolean value?
+2. Which JavaScript operators will return a Boolean value? 
+  <!-- Your answer: 
+        True or False conditions  -->
 
-  Your answer:
-
-  Researched answer:
+  <!-- Researched answer:
+        Logical operators (and, or, not, and not) -->
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  <!-- Your answer:
+        Index is a position in an array that starts at 0.  An index starts with (i) and value starts with (v). (Dad joke for Elyse)  An index is giving the position to something in a paramater and the value is giving the overall quantity within the given parameter.   -->
 
-  Researched answer:
+  <!-- Researched answer:
+        An index is the position of an expression in a string.   -->
 
 
 
 4. What is iteration?
 
-  Your answer:
+  <!-- Your answer:
+        Iteration is repeating lines of code. -->
 
-  Researched answer:
+  <!-- Researched answer:
+        Iteration is process of repeating steps. -->
 
 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-  Your answer:
+  <!-- Your answer:Driver/Navigator.  
+        Driver is typing the code per instruction from the Navigator who is giving and proofing the typed code. -->
 
-  Researched answer:
+  <!-- Researched answer:
+        Two programmers work together at one wokstation.  The driver writes the code while the navigator reviews the written code for errors then both parties switch roles.    -->
```

Comment:
While your answers in this section are accurate, they are meeting the bare minimum of the requirements for this section.  In an interview, answering with short, one liners will come off as uninterested and will not bring you success with most interviews.  I would really like to see more thorough answers next week providing as much detail and even examples when necessary.